### ISSUE TITLE
Still functional?

### ISSUE THREAD

**Issue by inspiredearth** [Aioros/livehosts#1]
Hi there,
Does this Chrome extension actually still work on latest versions of Chrome?
It seems to have no effect at all, in my browser.
Thanks.

Jonathan

**Comment by Aioros** [comment]
Hi Jonathan, is it possible you have an old version? I had to remove and re-upload the extension on the chrome store a while ago. The current version is here: https://chrome.google.com/webstore/detail/livehosts/hdpoplemgeaioijkmoebnnjcilfjnjdi

Could you reinstall it from there and let me know if anything changes?

**Comment by inspiredearth** [comment]
Hi Stefano,

Thanks for the prompt reply. Greetings from New Zealand.

I went to that page, and seems to be where I installed it from.

However, I reinstalled it from there, and this time it works. So that’s great.

Thanks very much for this plug-in. It’s going to save me a lot of mucking around.

Regards,

J

**Comment by inspiredearth** [comment]
Now that it's working, there's another issue I see.
Even if I untick both "This tab" and "All tabs", the IP change will continue to take effect.
I have to completely remove the relevant IP change entry.
Is that intentional? I get the impression the idea you had was that the use can activate and deactivate the IP mappings, without having to remove the entries.

**Comment by Aioros** [comment]
Definitely not intentional. Thanks for letting me know, I'll look into it.

**Comment by inspiredearth** [comment]
Great. Would love to see this working well, as it's such a time-saver / annoyance reducer. :-)

**Comment by inspiredearth** [comment]
I also note that it crosses the barrier between Private browser session and the regular sessions.

So if I activate a livehosts rule in a private window, it will affect the non-private windows.

**Comment by Aioros** [comment]
Ok, that's great information too, thanks. I'll keep you updated.

**Comment by Aioros** [comment]
I just uploaded an updated version that should fix the This tab/All tabs checkboxes. The private session separation might take some more time, I opened issue #2 to track it.
The extension should update automatically for you at a certain point (or you could force the update), let me know if it's fixed for you when it does.

EDIT: Actually, I'm going to wait on publishing that change until I solve issue #2 too, because LiveHosts always ends up requiring a manual review that takes a few days.

EDIT2: The updated version (with both issues fixed, hopefully) is undergoing the review, let's hope it takes less than a week.

**Comment by Aioros** [comment]
Version 1.0 is now available in the store. @jonathannz , if you have the chance to try it, let me know if the issue is solved for you. Thanks.

**Comment by inspiredearth** [comment]
Thanks for letting me know. I’ll test it out shortly.

**Comment by inspiredearth** [comment]
Brilliant. It now works as expected. I’ve simply tested changing the IP/Host on one tab. It worked on that tab, and didn’t affect other tabs.
Also, I see that it doesn’t pass over to Incognito mode, and vice versa.

As a suggestion… personally, I’d prefer if it defaulted to being active for “This tab” only, and then clicking “All tabs” if such broad application is required.

**Comment by Aioros** [comment]
Good, thanks. Regarding the default, that probably makes sense. I think in the next release I'll provide a simple options page for similar choices.

**Comment by inspiredearth** [comment]
After a little more testing, I see that it has no effect in Incognito windows. So it’s at least not crossing the boundaries to affecting normal windows, but it’s not functional in the Incognito workspace.

**Comment by Aioros** [comment]
That's a bit weird. I would suggest you to try removing and recreating all existing rules, but I can imagine that would be a nuisance. I have version 1.1 pending review in the store at the moment, which has a few important fixes, so hopefully it will work better.

(By the way, sorry about the inconvenience. I developed this extension mostly as an exercise for myself, and I don't have a lot of resources for testing. Your feedback has been very useful.)
